examples/michael_schumacher_problem/main_michael_schumacher.m

```diff
@@ -150,9 +153,16 @@
 % intitial guess for fun
 alpha = [atan2(diff(yy),diff(xx)),0];
 x_guess = vertcat(xx,yy, zeros(2,problem_options.N_stages), alpha);
+
+active_set_guess = nosnoc.activeset.Pss({[1,2]},'times', [problem_options.T]);
+solver_options.mpecopt.initialization_strategy = 'TakeProvidedActiveSet';
 %% Solve
 ocp_solver = nosnoc.ocp.Solver(model, problem_options, solver_options);
-ocp_solver.solve();
+
+ocp_solver.set_initial_active_set(active_set_guess);
+[IG,IH,I00] = ocp_solver.discrete_time_problem.process_active_set(active_set_guess);
+
+ocp_solver.solve('mpecopt', IG=IG, IH=IH);
```

Comment:
so this give an error message of too many input arguments, on the other hand, removing this leads to  a warning:

Warning: \n TakeProvidedActiveSet as initialization strategy chosen, but no y0 provided. Resporting to TakeInitialGuessActiveSet 

seems like the passing does not quite work.

Reply:
also, there is a typo in the warning message :D

Reply:
That warning is in mpecopt :sweat_smile:. This is a bug caused by some rebasing, it has been fixed.